EjercicioDiaDos.cs

```diff
@@ -37,52 +35,39 @@ public void llamarEjercicioUno()
         public void llamarEjercicioDos()
         {
             Console.WriteLine("Digite un Número: ");
-            int numero = int.Parse(Console.ReadLine());
-
-            EjercicioDiaDos ejercicioDia = new EjercicioDiaDos();
+            int numero = Convert.ToInt32(Console.ReadLine());
 
-            numero = ejercicioDia.DeterminarNumero(numero);
+            string determinenumero = ejercicioDia.DeterminarNumero(numero);
 
-                Console.WriteLine("Número es Negativo" + (numero < 0));
-                Console.WriteLine("Número es Positivo" + (numero >= 0));
+            if (numero >= 0)
```

Comment:
Estos condicionales no deben de ir acá: Tu vas a llamar un metodo que te va a devolver un string con: positivo o con negativo y luego haces  un

```suggestion
          Console.WriteLine("Número es: "  + determinenumero));
```

Reply:
Cambios realizados a punto

Reply:
Realizado